azext_iot/tests/central/json/device_template_int_test.json

```diff
@@ -8,6 +8,15 @@
         "@type": [
```

Comment:
does this file also need the "@type": "CapabilityModel" change?

Reply:
I see this got resolved, but I don't see the updates, was this a won't fix? Please only resolve the comment after updating the code, or after providing a #wontfix with a reason